__tests__/profile/change-profile.test.tsx

```diff
@@ -0,0 +1,100 @@
+// import React from 'react'
```

Comment:
I'd like to kindly remind you to not comment out your tests. We aim to maintain a minimum test coverage of 80%. If there are issues with the tests, let's focus on resolving them rather than commenting them out! 😄

Reply:
Thx for the reminder ive added more tests with > 85% coverage and all passed